### PR TITLE
add: autoware labs logo

### PR DESCRIPTION
**Pull request by mozhoku**
## Description

This PR adds Autoware Labs logo and moves the logos on the screen to MainMenuCanvas. Now logos and AWSIM version can be seen by pressing the "esc" key

### Related links

- https://github.com/autowarefoundation/AWSIM/issues/35

## Tests performed

Not applicable.

## Effects on system behavior

It does not affect system behaviour.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by mozhoku** [comment]
Logos can be seen in the bottom left corner.
image

**Comment by xmfcx** [comment]
could you share some screenshots with this change?

File: Assets/AWSIM/Scenes/Main/AutowareSimulation.unity [hunk 1]
```diff
@@ -1868,7 +2024,7 @@ MonoBehaviour:
     m_HorizontalOverflow: 0
     m_VerticalOverflow: 0
     m_LineSpacing: 1
-  m_Text: AWSIM  v 0.0.0
+  m_Text: AWSIMLabs  v 0.0.0
```
Comment (xmfcx) [inline Assets/AWSIM/Scenes/Main/AutowareSimulation.unity]:
```suggestion
  m_Text: AWSIM Labs  v 0.0.0
```

File: Assets/AWSIM/Scripts/UI/DemoUI.cs [hunk 2]
```diff
@@ -19,7 +19,7 @@ private void Start()
             timeScaleText.text = "x " + timeScaleSlider.value.ToString("F2");
             var version = Application.version;
             print(version);
-            versionText.text = "AWSIM v " + version;
+            versionText.text = "AWSIMLabs v " + version;
```
Comment (xmfcx) [inline Assets/AWSIM/Scripts/UI/DemoUI.cs]:
```suggestion
            versionText.text = "AWSIM Labs v " + version;
```

**Comment by mozhoku** [comment]
> could you share some screenshots with this change?

There is already a screenshot above.

**Comment by xmfcx** [comment]
Could you also add a white Autoware Foundation logo and the "The Autoware Foundation" text?

https://autoware.org/media-kit/

**Comment by mozhoku** [comment]
Added the Autoware Foundation logo:
image

**Comment by xmfcx** [comment]
Could you move the company logos to the top left corner, in 2 rows?

And on the bottom left corner:
```
AWSIM Labs
v 1.0.0
```
near the Labs logo.

**Comment by xmfcx** [comment]
Could you also rebase to the main?

**Comment by mozhoku** [comment]
> Could you move the company logos to the top left corner, in 2 rows?
> 
> And on the bottom left corner:
> 
> ```
> AWSIM Labs
> v 1.0.0
> ```
> 
> near the Labs logo.

image

Something like this?

**Comment by xmfcx** [comment]
Top left:
- The company logos should be relatively same size and aligned

Bottom left:
- The logo should be on the left side of the text

All items:
- All items should have a set margin from the edges of the window.
- Margin could be something like 20px? I need to see.

**Comment by mozhoku** [comment]
> Top left:
> 
> * The company logos should be relatively same size and aligned
> 
> Bottom left:
> 
> * The logo should be on the left side of the text
> 
> All items:
> 
> * All items should have a set margin from the edges of the window.
> * Margin could be something like 20px? I need to see.

I've set the margins something around 20px here:

image

**Comment by xmfcx** [comment]
320067127-a2d0c987-ec33-4006-8f5e-8daa81fb87c6

**Comment by mozhoku** [comment]
I'll try my best!

I thought scaling the Autoware Foundation logo to Tier IV's would be enough.

**Comment by mozhoku** [comment]
Okay done!
image

File: Assets/AWSIM/Scripts/UI/DemoUI.cs [hunk 3]
```diff
@@ -19,7 +19,7 @@ private void Start()
             timeScaleText.text = "x " + timeScaleSlider.value.ToString("F2");
             var version = Application.version;
             print(version);
-            versionText.text = "AWSIM v " + version;
+            versionText.text = "AWSIM Labs v " + version;
```
Comment (xmfcx) [inline Assets/AWSIM/Scripts/UI/DemoUI.cs]: Could you make this just the version text?

And AWSIM Labs will be a separate text entity.

So we can color it later on.

Comment (mozhoku) [reply Assets/AWSIM/Scripts/UI/DemoUI.cs]: To make it a separate text entity I would have to create a text object in the canvas.

**Review by xmfcx — APPROVED** [review]

**Pull request merged**